eo-maven-plugin/src/test/java/org/eolang/maven/Xmir2Xmir.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2024 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import com.jcabi.xml.XMLDocument;
+import com.yegor256.Jaxec;
+import com.yegor256.xsline.Shift;
+import com.yegor256.xsline.TrClasspath;
+import com.yegor256.xsline.Xsline;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import org.cactoos.Scalar;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.scalar.Unchecked;
+import org.cactoos.text.TextOf;
+import org.eolang.maven.util.HmBase;
+import org.eolang.maven.util.Home;
+import org.yaml.snakeyaml.Yaml;
+
+/**
+ * Test scenario in YAML.
+ *
+ * @since 0.1
+ * @todo #3267:90min Add better error printing. Ideally, it should print the difference
+ *  the way the linux diff command does (line by line).
+ */
+@SuppressWarnings({"JTCOP.RuleAllTestsHaveProductionClass", "JTCOP.RuleCorrectTestName"})
+public final class Xmir2Xmir {
```

Comment:
@levBagryansky may this class be package private?

Reply:
@maxonfjvipon done